docs/modules/ROOT/partials/install/prepare-terraform.adoc

```diff
@@ -12,6 +12,15 @@ mkdir -p inventory/classes/
 git clone $(curl -sH"Authorization: Bearer ${COMMODORE_API_TOKEN}" "${COMMODORE_API_URL}/tenants/${TENANT_ID}" | jq -r '.gitRepo.url') inventory/classes/${TENANT_ID}
 ----
 
+. Set team responsible for handling Icinga alerts
```

Comment:
Do we need to specify the technology here?
```suggestion
. Set team responsible for handling alerts
```

Reply:
I'd leave it, since this variable is only used to configure the Puppet-based monitoring for the LBs.